src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/feedback/feedback.component.spec.ts

```diff
@@ -0,0 +1,5 @@
+import _ from 'lodash';
+
+describe('FeedbackComponent', () => {
+  
+});
```

Comment:
tests need to test at least whether the component is created. (but you can work on it once everything else is completed too)